pkg/sql/distsql_plan_changefeed.go

```diff
@@ -163,10 +163,6 @@ func PlanCDCExpression(
 		return cdcPlan, errors.AssertionFailedf("expected at least 1 span to scan")
 	}
 
-	if len(presentation) == 0 {
```

Comment:
I'm hesitant to simply delete this check because it would prevent us from catching a developer error in the vast majority of cases when the table isn't empty. I think we might want to check a more specific condition instead, although I don't currently know what that should be. I'll consult with my team to see if they have any ideas.

Reply:
Thanks. I'd appreciate any tips

Reply:
Hey @andyyang890 Any updates?